components/forms/Button/Button.tsx

```diff
@@ -11,24 +10,12 @@ interface ButtonProps {
   disabled?: boolean;
   destructive?: boolean;
   testid?: string;
-  ariaProps?: React.AriaAttributes;
 }
 
 export const Button = (
   props: ButtonProps & JSX.IntrinsicElements["button"]
 ): React.ReactElement => {
-  const {
-    type,
-    children,
-    secondary,
-    base,
-    onClick,
-    className,
-    testid,
-    destructive,
-    disabled,
-    ariaProps,
-  } = props;
+  const { children, secondary, base, onClick, className, testid, destructive } = props;
```

Comment:
Looking at the failing test ... 
 
 we should use ...rest 
 
 { children, secondary, base, onClick, className, testid, destructive, ...rest } = props;
 
 ```
 <button
      className={classes}
      onClick={onClick}
      data-testid={testid ? testid : "button"}
      {...rest}
    >
 ```